doc/specifications/drafts/content_edit.md

```diff
@@ -0,0 +1,29 @@
+# Symfony forms content edit
+
+## Synopsis
+repository-forms currently supports creating content without an intermediate draft.
+
+Support must be added for editing content. It implies creating a new version of an existing content,
+rendering a form with the previous version's fields values, and allowing the user to save the draft,
+cancel or publish the new version. It should also allow choosing a language, either for translation
+or 
+
+Ideally, the draft would only be created when we need it, not on the first display.
```

Comment:
I would define the requirements like:
- Initial version of the object (v0) should not require a draft to be created before store.
- Edits of existing versions should create a draft as it is needed for checking if it is already being worked on etc. as AR mentions.

The overal idée is to not create a draft until you actually publish something. This is mainly to remove a potential DOS attack and also create many "loose drafts" whenever users are e.g. registering. Can easily be exploited to fill up the DB.